### PR TITLE
🚚 (service engineer)- fixed paths, called get methods

### PR DESCRIPTION
**Pull request by thangeniFT**
edited files

### CODE REVIEW

File: libs/api/upintegration/service/feature/src/lib/queries/handlers/getStudentInfo.handler.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { IQueryHandler, QueryHandler } from "@nestjs/cqrs";
+import { UpIntegrationRepository  } from "@graduates/api/upintegration/repository/data-access";
+import {getStudentInfoEmailsQuery,getStudentInfoNameQuery,getStudentInfoFilesQuery } from "../impl";
+
+@QueryHandler(getStudentInfoNameQuery)
+export class getStudentInfoNameHandler implements IQueryHandler<getStudentInfoNameQuery> {
+    constructor(private readonly repository: UpIntegrationRepository) {}
+
+    async execute(query: getStudentInfoNameQuery): Promise<any> {
```
Comment (u20437863-James-Butler) [inline libs/api/upintegration/service/feature/src/lib/queries/handlers/getStudentInfo.handler.ts:9]: This is just giving a warning about the type being any. Is this intended?

File: libs/api/upintegration/service/feature/src/lib/upintegration-service.module.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { ApiUpintegrationRepositoryDataAccessModule } from '@graduates/api/upintegration/repository/data-access';
+import { Module } from '@nestjs/common';
+import { CommandBus, CqrsModule, QueryBus } from '@nestjs/cqrs';
+import { ApiupintegrationService } from './api-upintegration.service';
+import * as QueryHandlers from './queries/handlers';
```
Comment (u20437863-James-Butler) [inline libs/api/upintegration/service/feature/src/lib/upintegration-service.module.ts:5]: Two libraries are imported that are never used, is this setup for future code?

**Review by u20437863-James-Butler — APPROVED** [review]
I'm happy with the additions here, just confirm that those warnings aren't a problem and I'll merge

**Comment by thangeniFT** [comment]
I did not getting a warning on my side, I am however looking at it again now

File: libs/api/upintegration/service/feature/src/lib/queries/handlers/getStudentInfo.handler.ts (hunk 1 above)
Comment (thangeniFT) [reply libs/api/upintegration/service/feature/src/lib/queries/handlers/getStudentInfo.handler.ts:9]: The warning is not a problem

File: libs/api/upintegration/service/feature/src/lib/upintegration-service.module.ts (hunk 2 above)
Comment (thangeniFT) [reply libs/api/upintegration/service/feature/src/lib/upintegration-service.module.ts:5]: Yes

**Comment by u20437863-James-Butler** [comment]
Ok, if the warnings won't cause issues we can merge and fix it at a later stage.

**Pull request merged**